c2cwsgiutils/health_check.py

```diff
@@ -45,12 +53,17 @@ def json_data(self) -> Any:
         return self.json
 
 
-def _get_bindings(session: Any) -> List[sqlalchemy.engine.Engine]:
-    return (
-        [session.c2c_rw_bind, session.c2c_ro_bind]
-        if session.c2c_rw_bind != session.c2c_ro_bind
-        else [session.c2c_rw_bind]
-    )
+def _get_bindings(session: Any, engine_type: EngineType) -> List[sqlalchemy.engine.Engine]:
```

Comment:
```suggestion
def _get_bindings(session: Any, engine_type: EngineType=None) -> List[sqlalchemy.engine.Engine]:
```
and change the first If?

Reply:
Since the `get_bindings_ function is private I chose to make it mandatory to pass the engine information.
As a consequence, for backward compatibility, all calls to this function now pass `READ_WRITE` by default.